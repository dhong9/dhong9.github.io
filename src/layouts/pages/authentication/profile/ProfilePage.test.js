// Unit test libraries
import React from "react";
import renderer from "react-test-renderer";

// Component to test
import ProfilePage from "layouts/pages/authentication/profile";

// Mocks
jest.mock("pages/LandingPages/Profile", () => {
  const { forwardRef } = jest.requireActual("react");
  return {
    __esModule: true,
    default: forwardRef(() => <div>Profile</div>),
  };
});

describe("ProfilePage", () => {
  it("renders", () => {
    const component = renderer.create(<ProfilePage />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
