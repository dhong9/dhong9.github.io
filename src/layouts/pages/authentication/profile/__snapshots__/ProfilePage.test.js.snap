// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfilePage renders 1`] = `
<div>
  Profile
</div>
`;
